lix/source-code/fs/src/implementations/memoryFs.ts

```diff
@@ -183,24 +183,30 @@ export function createNodeishMemoryFs(): NodeishFilesystem {
 			})
 
 			const listener = (event: FileChangeInfo) => {
-				if (event.filename === null) {
+				const eventToForward = {
+					eventType: event.eventType,
+					filename: event.filename,
+				}
+
+				if (eventToForward.filename === null) {
```

Comment:
@martin-lysk  could you rename eventToForward to event and  just destructure the incoming event directly? into eventType, filename ertc?